### PR TITLE
Fixes #601 - Added 'Checklist' section

### LINKED ISSUE #601: Checklist administration
```[tasklist]
### Tasks
- [x] Base structure
- [x] Template
- [x] Remarks / Limitations
```

### PR DESCRIPTION
**Pull request by ralf401**

### CODE REVIEW

**Comment by ralf401** [comment]
Hey @dvuckovic, can you have a look here, too? Thank you!!

File: manage/checklist.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+Usage
+-----
+
+To activate/deactivate the feature, simply switch the toggle labeled with
+"Enable Checklists" at the top of the page.
+
+You can optionally create one or more templates with predefined checklist
+items. For this, just click on the "New Checklist Template" button and add
+the checklist items. After submitting the form, the template can be used in
```
Comment (dvuckovic) [inline manage/checklist.rst:23]: Perhaps better: `...and define the checklist name and items.`

File: manage/checklist.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+tickets.
+
+You can clone or delete the template by clicking on the ︙ button and select
+the corresponding action.
```
Comment (dvuckovic) [inline manage/checklist.rst:27]: `the template` => `a template`
`select` => `selecting`

File: manage/checklist.rst [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+To edit the template, simply click on the title of the template. This opens
```
Comment (dvuckovic) [inline manage/checklist.rst:29]: `title` => `name`

File: manage/checklist.rst [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+   :alt: Screenshot showing the checklist template edit dialog.
+   :align: center
+   :scale: 70%
+
+Remarks
+-------
+
+- It is only possible to add one checklist per ticket.
+- There is no specific permission for the checklist feature. You can turn it off
+  or on and provide optionally usable checklist templates.
```
Comment (dvuckovic) [inline manage/checklist.rst:43]: Perhaps add something along the lines:

`If the agent has access to the ticket, they will also have access to its checklist.`

File: manage/checklist.rst [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+.. figure:: /images/manage/checklist/checklist-template-edit.png
+   :alt: Screenshot showing the checklist template edit dialog.
+   :align: center
+   :scale: 70%
+
+Remarks
+-------
+
+- It is only possible to add one checklist per ticket.
+- There is no specific permission for the checklist feature. You can turn it off
```
Comment (dvuckovic) [inline manage/checklist.rst:42]: Do we need to also mention that customers have no access to the ticket checklist?

**Review by dvuckovic — APPROVED** [review]

**Pull request merged**